apps/src/templates/projects/ReportAbusePopUp.jsx

```diff
@@ -0,0 +1,279 @@
+          checked: false,
+        },
+        {
+          key: 'Offensive-Content',
+          label: i18n.abuseTypeOffensive(),
+          checked: false,
+        },
+        {
+          key: 'Copyright-Infringement',
+          label: i18n.abuseTypeInfringement(),
+          checked: false,
+        },
+        {key: 'Other', label: i18n.abuseTypeOther(), checked: false},
+      ],
+      showRecaptcha: false,
+      submitButtonEnabled: false,
+      loadedCaptcha: false,
+    };
+    this.cancel = this.cancel.bind(this);
+    this.onCaptchaVerification = this.onCaptchaVerification.bind(this);
+    this.token = ''; // captcha token
+    this.onCaptchaExpiration = this.onCaptchaExpiration.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleCheckboxChange = this.handleCheckboxChange.bind(this);
+  }
+
+  componentDidUpdate() {
+    const {showRecaptcha, loadedCaptcha} = this.state;
+
+    if (showRecaptcha && !loadedCaptcha) {
+      this.createCaptchaScript();
+    }
+
+    if (!showRecaptcha && loadedCaptcha) {
+      this.cleanUpCaptcha();
+    }
+  }
+
+  componentWillUnmount() {
+    this.cleanUpCaptcha();
+  }
+
+  cleanUpCaptcha() {
+    const captchaScript = document.getElementById('captcha');
+    if (captchaScript) {
+      const adjacentDiv = captchaScript.nextElementSibling;
+      adjacentDiv.remove();
+      captchaScript.remove();
+    }
+    this.setState({loadedCaptcha: false});
+  }
+
+  cancel() {
+    this.props.onClose();
+    this.setState({
+      captchaCompleted: false, // reset captcha completion
+      showRecaptcha: false, // reset checkboxes
+      checkboxes: [
+        {
+          key: 'Cyberbullying',
+          label: i18n.abuseTypeCyberbullying(),
+          checked: false,
+        },
+        {
+          key: 'Offensive-Content',
+          label: i18n.abuseTypeOffensive(),
+          checked: false,
+        },
+        {
+          key: 'Copyright-Infringement',
+          label: i18n.abuseTypeInfringement(),
+          checked: false,
+        },
+        {key: 'Other', label: i18n.abuseTypeOther(), checked: false},
+      ],
+    });
+    this.cleanUpCaptcha();
+  }
+
+  onCaptchaVerification(token) {
+    this.setState({
+      submitButtonEnabled: true,
+      captchaCompleted: true,
+    });
+    this.token = token;
+  }
+
+  handleSubmit() {
+    this.sendReportAbuse();
+  }
+
+  sendReportAbuse() {
+    const formData = new FormData();
+
+    // name, email, age set in controller through current_user
+    formData.append(
+      'authenticity_token',
+      RailsAuthenticityToken.getRailsCSRFMetaTags().token
+    );
+    formData.append('channel_id', this.props.projectData.channel);
+    formData.append('abuse_url', this.props.abuseUrl);
+    formData.append('g-recaptcha-response', this.token);
+
+    const checkedCheckboxNames = this.state.checkboxes
+      .filter(checkbox => checkbox.checked)
+      .map(checkbox => checkbox.key.replace(/-/g, ' '));
```

Comment:
Seems like the abuse report types we'll be reporting will be marginally different than what gets reported to Zendesk currently via the normal report abuse form (eg, "Copyright Infringement" instead of "infringement"). Probably fine, but just noting in case it becomes an issue (eg, if support has any metrics that rely on these specific strings).